software/soapysdr-xtrx/XTRXDevice.cpp

```diff
@@ -226,6 +226,7 @@ SoapyXTRX::SoapyXTRX(const SoapySDR::Kwargs &args)
         _dma_buf = NULL;
         break;
     case TargetDevice::GPU:
+#ifdef CUDA
```

Comment:
This is not finished... I need to plumb through the Cmake file to define this somehow.